### DISCUSSION TITLE
GPU enablement

### DISCUSSION THREAD

**Discussion by agbarbosa** [tairov/llama2.mojo#84] [General]
Hi @tairov, this is an amazing initiative! I am wondering when the code will allow GPU processing triggered by parameters. Do you have any plans for it?

**Comment by agbarbosa** [comment]
By going through the Modular / MAX documentation, just learned GPU capability is still not available for Mojo, right?
Reference: https://docs.modular.com/assets/images/max-stack-4ef999863182067c98e7292ed9e896d4.png#light

**Comment by tairov** [comment]
Hi @agbarbosa , so far Mojo and MAX don't support GPU execution. Once support will be added to Mojo we can adapt llama2.mojo for GPU inference.
